conf/server.yaml.template

```diff
@@ -65,3 +65,4 @@ SERVER:
     COMMAND_TIMEOUT: 300000
     # Time to wait for establishing the ssh connection, in seconds
     CONNECTION_TIMEOUT: 60
+  DEFAULT_OS_SEARCH_QUERY: 'name="RedHat" AND (major="6" OR major="7" OR major="8" OR major="9")'
```

Comment:
Why `major="6" OR major="7" OR major="8" OR major="9"`? What about 10? What about potential future versions? Why for 6..9?

Reply:
I'm ok with that.

Reply:
@dosas Firstly, I’m not quite clear on the motivation behind moving the search string from constants to the config.
And, I agree with @ogajduse here, as this approch wouldn't help us to dynamically generate rhel versions, instead IMO moving the search string from constants to config could create complexities/confusion between the sat_version  and rhel_version it supports, as configs are global for all sat_versions. For example, 6.17+ supports RHEL10 clients, but not earlier versions, where this searching for EL10 would be unnesessary .

Reply:
I dont see this as a good fit for `server.yaml` - it doesn't relate to Satellite

I would introduce `os.yaml`